js/client/modules/@arangodb/testutils/instance-manager.js

```diff
@@ -622,6 +622,24 @@ class instanceManager {
     return JSON.parse(req["body"])[0];
   }
 
+  removeServerFromAgency(serverId) {
+    // Make sure we remove the server
+    for (let i = 0; i < 10; ++i) {
+      const res = arango.POST_RAW("/_admin/cluster/removeServer", JSON.stringify(serverId));
+      if (res.code === 404 || res.code === 200) {
+        // Server is removed
+        return;
+      }
+      // Server could not be removed, give supervision some more time
+      // and then try again.
+      require("internal").wait(0.2);
```

Comment:
please at least print a `.` so we see a loop is in progress.

Reply:
Done